locomotion-sdk/src/Components/PhoneNumberInput/codes.json

```diff
@@ -1,982 +1,982 @@
 [
```

Comment:
Why aren't we using a lib that gives us the flags?

Reply:
we are, the file was already there, im just using it now to get the code based on the default country